scripts/common/rook.sh

```diff
@@ -368,6 +368,67 @@ function rook_operator_ready() {
     return 0
 }
 
+function rook_is_healthy_to_migrate_from() {
```

Comment:
See that this check is a duplication of rook_is_healthy_to_upgrade which now is called in the migration process ONLY when we are moving from Rook